### ISSUE TITLE
Windows - WGL extensions require a dummy window that generates WM_DESTROY

### ISSUE THREAD

**Issue by ostrichlabs** [ostrichlabs/canary#2]
Old Work Item 2. From Azure DevOps:

"To retrieve WGL extensions, a dummy window and DC must be created. Then the function pointers can be retrieved and the dummy window destroyed. Doing so generates a WM_DESTROY message, which could be a false signal to end the program. This hasn't been a problem for some time now, but the Impediment remains as a note for future reference. It may also be necessary to revisit when I add the ability to change window sizes in future iterations of canary."

tl;dr learn how to properly Window in Windows

**Comment by ostrichlabs** [comment]
I'm a dummy and re-used the main WndProc() function for the dummy window. Need to use a dummy WndProc() function specific to the display module.

**Comment by ostrichlabs** [comment]
Two-minute fix in commit e2b4051.
